src/teleop/gui/src/components/Menu.vue

```diff
@@ -20,7 +20,7 @@
         <MenuButton link="#/ISHTask" name="ISH GUI" />
         <MenuButton link="#/SATask" name="Sample Acquisition GUI" />
         <MenuButton link="#/AutonTask" name="Autonomy Mission" />
-        <MenuButton link="#/Control" name="Temp Controls GUI" />
```

Comment:
Can we change the name of the route and file name to #/ESWDebug and ESWDebugGUI.vue respectively?